### PR TITLE
CLDR-17434 fix markdown lint warnings

### PR DESCRIPTION
**Pull request by DavidLRowe**
CLDR-17434

- [ ] This PR completes the ticket.

[Later edit] It seems that separate action will be needed to shorten the lines for the CLA section, as the source for that is elsewhere.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by DavidLRowe** [comment]
gh-pages.yml workflow changed to avoid running markdown linter on two .md files in .github folder.
Changes made to CONTRIBUTING.md to shorten lines and fix two broken links, using the proposals in the ticket.
One other change ("work" to "working") also made.
Targeting v46

**Comment by DavidLRowe** [comment]
I'm confused. I got an email saying the run had failed, but this summary says "All checks have passed".

Do you know where the markdownlint-cli warnings would appear (though hopefully there won't be any left)?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,46 +65,81 @@ begin with the accepted Jira ticket number.
...
-
-- **Individual CLA**: If you have determined that the Individual CLA is appropriate, then when you access the CLA Form, click the Individual CLA and complete the Form.
-
-- **Corporate CLA**: If you have determined that a Corporate CLA is appropriate, please first check the [public list of Corporate CLAs][unicode-corporate-clas] that the Consortium has on file. If your employer is listed, then when you access the CLA Form, you can click the box indicating that you are covered by your employer’s corporate CLA. If your employer is not on the list, then it has not already signed a CLA and you will need to arrange for your employer to do so before you contribute, as described in [How to Sign a Unicode CLA][signing].
-
-Unless otherwise noted in the [`LICENSE`](./LICENSE) file, this project is released under the [OSI-approved][osi-Unicode-License-3.0] free and open-source [Unicode License v3][unicode-license].
+In order to contribute to this project, the Unicode Consortium must have on
```
Comment (srl295) [inline CONTRIBUTING.md:68]: Please don't edit the "Contributor License Agreement" section. It is from the licensing team and is legally significant. I would revert this section and we can take a separate action to update this text up-stream.

**Review by srl295 — CHANGES_REQUESTED** [review]
the "Contributor License Agreement" part of CONTRIBUTING.md may not be updated here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (DavidLRowe) [reply CONTRIBUTING.md]: @srl295 This change was reverted. Ready for re-review.

I also edited the comment at the top to indicate that this PR won't close the ticket.

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
@DavidLRowe i'll add a comment on the ticket

**Pull request merged**